gems/http/5.1/http.rbs

```diff
@@ -0,0 +1,47 @@
+# Write the type definition here!
+
+module HTTP
+  interface _Stringable
+    def to_s: () -> String
+  end
```

Comment:
[suggest] I'm unsure but the built-in `_ToS` interface may be available instead:

https://github.com/ruby/rbs/blob/v3.1.1/core/builtin.rbs#L13-L15

Reply:
@ybiquitous Thank you! I fixed at 3b6d7cb